specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2021-09-01-preview/appplatform.json

```diff
@@ -3508,6 +3516,21 @@
         }
       }
     },
+    "LoadedCertificate": {
+      "description": "Loaded certificate payload",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Resource Id of loaded certificate",
+          "type": "string"
+        },
+        "loadTrustStore": {
```

Comment:
Shall we consider a parent level jobject to wrap this into it. I'm more considering in future we might add "loadKeyStore" for Java and some other stuff for .net.